stable/mission-control/README.md

```diff
@@ -461,8 +461,7 @@ The following table lists the configurable parameters of the mission-control cha
 | `insightServer.name`                         | Insight Server name                             | `insight-server`                      |
 | `insightServer.image.repository`             | Container image                                 | `docker.bintray.io/jfrog/insight-server`|
 | `insightServer.image.version`                | Container image tag                             | `.Chart.AppVersion`                   |
-| `insightServer.externalHttpPort`             | Insight Server service external port            | `8082`                                |
```

Comment:
Why did you remove the external port?

Reply:
we don't need this - Everything is via router .

Reply:
👍